Mathlib/RingTheory/Artinian.lean

```diff
@@ -267,15 +306,6 @@ theorem disjoint_partial_infs_eventually_top (f : ℕ → Submodule R M)
   exact (h m).eq_bot_of_ge (sup_eq_left.1 <| (w (m + 1) <| le_add_right p).symm.trans <| w m p)
 #align is_artinian.disjoint_partial_infs_eventually_top IsArtinian.disjoint_partial_infs_eventually_top
 
-lemma _root_.LinearMap.exists_range_pow_eq_iInf (f : Module.End R M) :
```

Comment:
This is removed because it has been replaced by the more powerful (and more useful) lemma: `LinearMap.eventually_iInf_range_pow_eq` (above).